wm-paper/rfc/index.md

```diff
@@ -0,0 +1,91 @@
+# Stable UI
+
+After a lot of learnings, we can be proud that we have figured out what Watermelon will look like for the short and medium term. We have a stable UI that we can use to build the extension.
```

Comment:
This is not an entirely true statement.

Startups always have this tradeoff between focus VS iteration. We've agreed on a lot of statements being true, which gives us a long-term north. We've also proved and disproved various hypotheses which have given shape to our product. But we have to de-risk other hypotheses first, before entirely setting. 

- Will companies self serve and pay for what we currently have? (riskiest hypothesis at the moment)
- What if we try the same experiment, but with a Slack integration?
- If not, do we need to build this as something of another shape (eg: GitHub Action). 

That's why startups who ship fast, without being perfect succeed. Because they de-risk stuff, and once they see it working they double down on what worked, and discard what didn't. Speed of iteration is the best predictor of wether a startup will achieve PMF or not